sources/packages/api/src/database/entities/institution.model.ts

```diff
@@ -72,12 +68,6 @@ export class Institution extends RecordDataModel {
   })
   institutionPrimaryContact: InstitutionPrimaryContact;
 
-  @Column({
-    name: "legal_authority_contact",
```

Comment:
Why it's removed, this is the one you added right, Not sure, if I am missing anything
![image](https://user-images.githubusercontent.com/77353155/158475095-d91404af-769f-498e-b8b5-e894582d3a01.png)

Reply:
@ann-aot the add is part of the rollback script 😉 
![image](https://user-images.githubusercontent.com/61259237/158481282-05bc13df-3afe-4a73-822b-5ce745a88990.png)

Reply:
Yes. Add is a part of rollback script.

Reply:
mm, file_name is confusing, can to rename the filename, maybe something like `rollback_add-legal-authority-contact.sql`